dnf5/commands/module/module_info.hpp

```diff
@@ -20,15 +20,19 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 #ifndef DNF5_COMMANDS_MODULE_MODULE_INFO_HPP
 #define DNF5_COMMANDS_MODULE_MODULE_INFO_HPP
 
+#include "module_list.hpp"
+
 #include <dnf5/context.hpp>
+#include <libdnf5/module/module_query.hpp>
```

Comment:
Minor: just that both `dnf5/context.hpp` and `libdnf5/module/module_query.hpp` are already included within the `module_list.hpp`.

Reply:
Thanks, removed.